src/components/navigation/navigation.html

```diff
@@ -18,6 +24,49 @@
       </div>
     </header>
     <main class="main-container">
+      <div class="respSidebar">
+        <span class="closeNav pr-10 pl-10 text-center" onclick="closeNav()">
+          Close <i class="far fa-times"></i>
+        </span>
+        <h3>Getting started</h3>
+        <nav class="intro-list">
```

Comment:
-try using same class name i.e., either camel-casing or - one.
`respSidebar` and `intro-list`